GOVERNANCE.md

```diff
@@ -98,6 +98,12 @@ If a vote is called, the following decision require Unanimity <https://en.wikipe
 - If a Maintainer of an affected git repository feels feedback from specific people is warranted they will @mention those users or teams to request feedback.
 - Proposals may also be added to the Flux Dev weekly meetings agenda, as a good avenue for making progress on a decision <https://lists.cncf.io/g/cncf-flux-dev/calendar>.
 
+### Examples
```

Comment:
I agree a few notable examples of past decision processes could be useful for future decision makers. And there are good examples. However this would probably be better as a separate file clearly linked from the main governance document. We have tried to keep this governance document as short and clear as posible to guide future decisions.

Reply:
I see this PR in the spirit of e.g. 189, 191, 192, 199. So not sure if this a "material change" - I see it more like an addendum and not an actual way of changed governance.

Reply:
I moved it go `community-roles.md`.